sql/create/release-1.0.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -113,9 +113,9 @@ CREATE TABLE `t_escheduler_master_server` (
   `host` varchar(45) DEFAULT NULL COMMENT 'ip',
   `port` int(11) DEFAULT NULL COMMENT 'port',
   `zk_directory` varchar(64) DEFAULT NULL COMMENT 'the server path in zk directory',
-  `res_info` varchar(256) DEFAULT NULL COMMENT 'json resource information:{"cpu":xxx,"memroy":xxx}',
+  `res_info` varchar(256) DEFAULT NULL COMMENT 'json resource information:{"cpu":xxx,"memory":xxx}',
```

Comment:
The length is best to be modified to 255

Reply:
I changed it.please review.